.readthedocs.yml

```
@@ -1,10 +1,13 @@
 version: 2
 
 build:
-  image: latest
+  os: "ubuntu-22.04"
+  tools:
+    python: "3.7"
```

Python 3.7 is EOL, can we update this?

```suggestion
    python: "3.11"
```

---

Took the liberty of applying this to see how it fares.